launcher/src/main/java/org/wildfly/core/launcher/AbstractCommandBuilder.java

```diff
@@ -587,6 +606,35 @@ protected Path normalizePath(final Path parent, final String path) {
         return serverArgs.asList();
     }
 
+    /**
+     * Returns the value of the {@code jboss.modules.os-name} property.
+     *
+     * @return the value of the {@code jboss.modules.os-name} property
+     */
+    public String getJBossModulesOSName() {
```

Comment:
I don't understand this patch.
It seems we are duplicating the logic to determine the `jbossModulesOSName` in both the scripts and here. Am I missing a difference?

Wouldn't it be enough to have only this `getJBossModulesOSName`method that will automatically set the field if the `os.name` correspond to RHEL 6-8?

Reply:
Also again, I am a bit worried that there will be a problem with WildFly distro executed on the RHELX platform - If I didn't miss anything then with this modification we fix this for EAP, although user of WildFly will now have to set `jboss.modules.os-name` with proper value in such case, right?

Reply:
> I don't understand this patch.
> It seems we are duplicating the logic to determine the `jbossModulesOSName` in both the scripts and here. Am I missing a difference?
> 
> Wouldn't it be enough to have only this `getJBossModulesOSName`method that will automatically set the field if the `os.name` correspond to RHEL 6-8?

Initially, I only modified standalone.sh and domain.sh. @jamezp had mentioned that we'd need launcher updates as well. @jamezp Did I misunderstand what you meant?

Reply:
@fjuma I just meant we should probably do what we're doing in the bash script here. I don't think we need to expose it, just add it as a JAVA_OPT, e.g. `addJavaOption("-Djboss.modules.os-name=el8)`.